Makefile

```diff
@@ -1,6 +1,16 @@
 REGISTRY ?= quay.io/kubevirt
 IMAGE_TAG ?= latest
 IMAGE_GIT_TAG ?= $(shell git describe --abbrev=8 --tags)
+PLATFORMS ?= linux/amd64,linux/s390x
```

Comment:
lets see please we are good to with having s390x by default, or better do the auto detection trick as done on cnao please
it is important to have fast deployment generally speaking imho